lib/support/mergeJSONGraphNode.js

```diff
@@ -192,7 +192,7 @@ module.exports = function mergeJSONGraphNode(
         }
     }
     else if (node == null) {
-        node = insertNode(message, parent, key, undefined, optimizedPath);
+        node = insertNode({}, parent, key, undefined, optimizedPath);
```

Comment:
i don't totally understand this part. what was `message` about, and why is it now replaced with an empty object?

Reply:
It looks like message is the current JsonGraphNode of the response JsonGraph.

Reply:
This change creates a new node instead of assuming that the node from the server is correctly formed. This prevents extra data stuffed into a node from the server from piggybacking into the cache.

Reply:
perfect, this solves the issue: "model.set() claims object supplied in arguments" as well!